### DISCUSSION TITLE
Last stable version

### DISCUSSION THREAD

**Discussion by mrouaux10** [nuxt/nuxt#21942] [Questions]
Hi, I will start a new project and i wanna use the last stable version available. Can I use Nuxt 3.6.1 or it's a beta version? Thanks!

**Comment by gerkereitz** [comment]
At the time of writing this answer, version 3.6.1 is the latest stable version available.

See more info at https://github.com/nuxt/nuxt/releases

**Reply by mrouaux10** [reply]
Great, thanks!
